### ISSUE TITLE
Unable to stop the worldping from pinging after removal

### ISSUE THREAD

**Issue by white-energy** [raintank/worldping-app#136]
Yeap... Those damn bots are spamming my logs, you should consider adding a fallback or somehow to stop this bug or whatever, thank you.

**Comment by woodsaj** [comment]
@white-energy you need to delete your endpoints before installing the app.

**Comment by Aamir010** [comment]
Same issue with me, it's keep pinging and hit to my https endpoint. Disable plugin from Grafana but still it's pinging. Can anyone please help me, how to disable endpoint or remove endpoint?

**Comment by woodsaj** [comment]
@Aamir010 the worldping-app is just the frontend to the service.  Removing the app from your grafana server does not affect the endpoints and checks you have running.  You need to delete the endpoints you created using the app.

**Comment by Aamir010** [comment]
Can you please help me, how to remove endpoint from that plugin

**Comment by woodsaj** [comment]
@Aamir010 if you send an email to support@raintank.io listing your grafana.net username/organisation and we can delete the endpoints for you.

Otherwise, you can do it through the app.
From the main menu in your Grafana:
- Select "worldping" -> "endpoints"
- for each endpoint, click the "details and config" (cog icon) on the very right
- navigate to the "configuration" tab
- expand the "Destroy" section at the botton and delete the endpoint.

**Comment by Dieterbe** [comment]
maybe it would be nice to have some sort of dialog when you try to remove the plugin that clears this up. cause i can see how this is confusing. it could say something like
"this won't remove the endpoints from being monitored.  if that's what you want, destroy the endpoints and then (optionally) remove this plugin if you no longer want it"

**Comment by woodsaj** [comment]
@Dieterbe the plugin is not removed through the UI.  It is removed by simlpy deleting the plugin dir from the grafana-server, or using grafana-cli.

**Comment by Dieterbe** [comment]
ok so we can discourage removing plugins by removing the directories, if plugins can specify callback messages then grafana-cli can display it when the user tries to delete the plugin

**Comment by Aamir010** [comment]
Yes, there is no such option on UI. But thanks @woodsaj Plugin gets deleted by grafana-cli.

**Comment by Aamir010** [comment]
I don't know what's going wrong, but still I am getting hits from worldping-api.

**Comment by white-energy** [comment]
As an alternative solution, I have used fail2ban to filter and ban those worldping bots

**Comment by woodsaj** [comment]
@Aamir010 did you remove the endpoints using the worldping-app, or did you just delete the worldping-app plugin.  As explained, you  need to delete the endpoints using the worldping-app before you remove the plugin from your grafana server.

If you have done all of this and are still having issues, then send an email to support@raintank.io with your grafana.net account name.

**Comment by white-energy** [comment]
Well, I have already uninstalled Grafana so reinstalling it just to disable endpoints is not worth it

**Comment by Aamir010** [comment]
@woodsaj There is no such thing find on Grafana UI, so I uninstall plugin. I send request to support@raintank.io but I don't get any confirmation. My application is still spamming

**Comment by willemdh** [comment]
Same problem here. Did a test with WOrldPing and although I disabled the WorldPing app in Grafana, it seems I'm still getiing pinged, resulting in a very high load on one of my servers. I also sent two emails to raintank, and I'm not getting any answer.

**Comment by mattttt** [comment]
Hi @willemdh, we'll take a look. Also, will try and hunt down your email now. Apologies that it seems to be going into the ether.

**Comment by willemdh** [comment]
@mattttt 

```
sudo netstat -anpt|grep httpd
tcp6       0      0 :::443                  :::*                    LISTEN      9942/httpd
tcp6      13      0 :::80                   :::*                    LISTEN      9942/httpd
tcp6       0      0 139.162.161.127:443     104.197.219.149:42175   ESTABLISHED 24469/httpd
tcp6       0      0 139.162.161.127:443     52.63.29.76:42617       ESTABLISHED 24471/httpd
tcp6       0      0 139.162.161.127:443     104.196.42.207:57385    ESTABLISHED 24467/httpd
tcp6       1      0 139.162.161.127:443     104.238.136.28:38311    CLOSE_WAIT  24398/httpd
tcp6       0      0 139.162.161.127:443     45.63.49.0:38610        ESTABLISHED 24462/httpd
tcp6       1      0 139.162.161.127:443     52.29.73.123:47139      CLOSE_WAIT  24320/httpd
tcp6       0      0 139.162.161.127:443     52.37.101.188:39661     ESTABLISHED 24468/httpd
tcp6       0      0 139.162.161.127:80      188.166.111.169:40126   ESTABLISHED 24472/httpd
tcp6       0      0 139.162.161.127:443     107.191.62.214:45796    ESTABLISHED 24466/httpd
tcp6       1      0 139.162.161.127:80      45.63.6.186:57222       CLOSE_WAIT  24321/httpd
tcp6       1      0 139.162.161.127:80      130.211.66.212:57531    CLOSE_WAIT  24473/httpd
tcp6       1      0 139.162.161.127:443     45.55.194.210:48344     CLOSE_WAIT  24395/httpd
tcp6       1      0 139.162.161.127:443     52.193.110.196:57304    CLOSE_WAIT  24456/httpd
tcp6       1      0 139.162.161.127:443     130.211.66.212:60194    CLOSE_WAIT  24453/httpd
tcp6       0      0 139.162.161.127:443     45.63.65.147:35681      ESTABLISHED 24459/httpd
tcp6       0      0 139.162.161.127:443     104.155.223.227:44699   ESTABLISHED 24470/httpd
tcp6       1      0 139.162.161.127:80      162.243.158.119:44628   CLOSE_WAIT  24079/httpd
tcp6       1      0 139.162.161.127:443     162.243.158.119:52142   CLOSE_WAIT  24396/httpd
tcp6       1      0 139.162.161.127:443     128.199.222.104:40551   CLOSE_WAIT  24461/httpd
tcp6       1      0 139.162.161.127:80      52.29.73.123:46775      CLOSE_WAIT  24214/httpd
tcp6       1      0 139.162.161.127:443     45.63.6.186:36824       CLOSE_WAIT  24376/httpd
tcp6       0      0 139.162.161.127:443     188.166.111.169:53960   ESTABLISHED 24455/httpd
tcp6       1      0 139.162.161.127:80      178.62.52.161:39740     CLOSE_WAIT  24387/httpd
tcp6       1      0 139.162.161.127:443     178.62.52.161:39798     CLOSE_WAIT  24256/httpd
tcp6       0      0 139.162.161.127:443     104.238.153.170:50416   ESTABLISHED 24460/httpd
```
Pinging the ip's from the 4th column always pints to vps cloud providers like Vultr, aws, Google, raintank..

`Pinging ams1.nl.collector.raintank.io [188.166.111.169] with 32 bytes of data:
Reply from 188.166.111.169: bytes=32 time=15ms TTL=55`

**Comment by nopzor1200** [comment]
Hi @white-energy and @willemdh sorry for the inconvenience. You two are absolutely right that we should have a fallback. We have created an easy way to stop all checks from worldPing; just log into grafana.net and you can disable all checks from the worldPing section.
